pyfar/dsp/dsp.py

```diff
@@ -227,6 +227,219 @@ def spectrogram(signal, dB=True, log_prefix=20, log_reference=1,
     return frequencies, times, spectrogram
 
 
+def time_window(signal, length=None, window='hann', shape='symmetric',
+                unit='samples', crop=True):
```

Comment:
I would go for `crop=False` as default, since a crop will change the length, and in case of a fade in also the linear-phase part of the signal/ir
```suggestion
                unit='samples', crop=False):
```

Reply:
Changed default to crop='none'